server/src/main/java/org/apache/cassandra/sidecar/acl/authorization/BasicPermissions.java

```diff
@@ -39,6 +39,9 @@
  */
 public class BasicPermissions
 {
+    // Auth cache related permissions
+    public static final Permission AUTH_CACHE = new StandardPermission("AUTH_CACHE", CLUSTER_SCOPE);
```

Comment:
Permissions are represented in format `resource:action`, I would suggest a `DomainAwarePermission` here with `CACHE:INVALIDATE`. We can reuse this permission for other caches too.